app/controllers/concerns/cookie_concern.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module CookieConcern
+  extend ActiveSupport::Concern
+
+  private
+
+  def set_default_cookies
+    return set_default_analytics_cookies if no_analytics_cookies?
+
+    if user_set_analytics_preference?
+      set_cookie(:analytics_cookies_set, value: params[:analytics_cookies_set])
+      set_cookie(:cookies_preferences_set, value: true)
+    end
+
+    set_analytics_cookies
+    show_hide_cookie_banners
+  end
+
+  def set_cookie(type, value: false)
```

Comment:
I would recommend making this a secure cookie.

Reply:
Thanks - I mistook this as being default, I've now reinstated it as secure